framework/levure.livecodescript

```diff
@@ -1867,7 +1879,7 @@ private command buildExtensions pArrayA, pOnlyBuildMissing
 end buildExtensions
 
 
-private command buildExtension pFilename, pOutputFolder
+private command buildExtension pFilenames, pOutputFolder
```

Comment:
Array variables need to be suffixed with an `A`.

Reply:
done